react/ui-components/src/atoms/HelpInfoCard.js

```
@@ -0,0 +1,330 @@
+
+  if (isSelectedFeatureLoading) return <Loader />;
+
+  const config = data?.[0] || {};
+
+  return (
+    <AlertCard
+      populators={{ name: "infocard" }}
+      variant="default"
+      className="cmn-help-info-card"
+      label={config?.title && t(config.title)}
+      additionalElements={[<AppHelpContent config={config}></AppHelpContent>]}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)